runtime/gc_base/accessBarrier.cpp

```diff
@@ -688,11 +688,10 @@ J9WriteBarrierJ9ClassBatchStore(J9VMThread *vmThread, J9Class *dstJ9Class)
 	barrier->preBatchObjectStore(vmThread, dstJ9Class);	
 }
 
-/* The only read barrier is Scavenger concurrent copy, pre read */
 void
 J9ReadBarrier(J9VMThread *vmThread, fj9object_t *srcAddress)
 {
-#if defined(OMR_GC_CONCURRENT_SCAVENGER)
+#if defined(OMR_GC_CONCURRENT_SCAVENGER) || defined(OMR_GC_VLHGC_CONCURRENT_COPY_FORWARD)
```

Comment:
In a way, even the original #ifdef was overkill. Probably added to minimize runtime cost of not calling virtual methods for platforms that do not support Concurrent Scavenger, but in OpenJ9 all platforms now support CS.

I'd say, just remove the #ifdefs.

Reply:
Fixed it! Thanks again